### PR TITLE
PLUGINS.md: update for `journal_entry_cqc`

### PR DESCRIPTION
**Pull request by norohind**
Maybe I should throw out at all from `state` table keys, that I know for sure will never appear with any meaningful values in CQC.
Also will be needed to correct first version EDMC where `journal_entry_cqc` is present now I stated it as `5.1.3+`.

### CODE REVIEW

File: PLUGINS.md [hunk 1]
```diff
@@ -737,6 +740,91 @@ Examples of this are:
    
 ---
 
+---
+
+### Journal entry CQC
+New in version 5.1.3+
```
Comment (Athanasius) [inline PLUGINS.md:746]: I'm targeting 5.2.0 for this, along with CAPI changes.

File: PLUGINS.md [hunk 2]
```diff
@@ -737,6 +740,91 @@ Examples of this are:
...
+```python
+def journal_entry_cqc(cmdr: str, is_beta: bool, entry: Dict[str, Any], state: Dict[str, Any]) -> None:
+    if entry['event'] == 'Location':
+        # We loaded to CQC match, lets detect map!
+        cqc_maps = {  # dict to map systems names to CQC maps
+        'Bleae Aewsy GA-Y d1-14': 'Asteria Point',
```
Comment (Athanasius) [inline PLUGINS.md:752]: The forum reference I know of for these is <https://forums.frontier.co.uk/threads/cqc-systems.234394/>, please add that to the comments here.

**Review by Athanasius — CHANGES_REQUESTED** [review]

**Comment by Athanasius** [comment]
> Maybe I should throw out at all from `state` table keys, that I know for sure will never appear with any meaningful values in CQC.

If you're 100% certain about them, then maybe.  On the other hand this is the same `monitor.state` dict being passed in as for `journal_entry()` and I'm more inclined to remove the table for the _cqc version in the docs and just point out that some of it will not be populated when in CQC.

Has someone checked that entering CQC, particularly from in-game, definitely leads to us resetting `monitor.state` to default/initial values ?

File: PLUGINS.md (hunk 2 above)
Comment (norohind) [reply PLUGINS.md:752]: But this thread doesn't contain such mapping, good news is I am posted it there.

**Comment by Athanasius** [comment]
> Has someone checked that entering CQC, particularly from in-game, definitely leads to us resetting `monitor.state` to default/initial values ?

Answer:  It's literal done in the same `LoadGame` handling that is detecting entering CQC.  So, yes, it'll be the correct state.

**Comment by norohind** [comment]
` I'm more inclined to remove the table for the _cqc version in the docs and just point out that some of it will not be populated when in CQC` that's what I meant.

**Review by Athanasius — APPROVED** [review]

**Pull request merged**